test/TestExpectations.json

```diff
@@ -47,6 +47,12 @@
     "parameters": ["webDriverBiDi"],
     "expectations": ["SKIP"]
   },
+  {
+    "testIdPattern": "[headful.spec] *",
+    "platforms": ["darwin", "linux", "win32"],
+    "parameters": ["firefox"],
+    "expectations": ["SKIP"]
```

Comment:
@Lightning00Blade Hm, what was the reason to skip all the tests for Firefox?

Reply:
https://github.com/puppeteer/puppeteer/pull/12037

Reply:
Thanks!